BlueprintUI/Sources/Layout/Builder.swift

```diff
@@ -23,6 +23,15 @@ public struct Builder<Child> {
         [child]
     }
 
+    @available(
+        *,
+        unavailable,
+        message: "Optionals must be unwrapped with `if let value = ...` to be returned from a result builder."
+    )
+    public static func buildExpression(_ child: Child?) -> Children {
+        fatalError()
```

Comment:
Does this work?
```suggestion
        guard let child = child else {
            return []
        }
        return child
```

Reply:
This code can never be reached though – so implementing this provides no value.

Reply:
Would it work if we removed `configure`? Or if `configure` is used, are you sure it doesn't disambiguate and then go down this path?

Reply:
If we remove `configure`, we'd have to go through and migrate everyone to the result builder syntaxes, which seems like a lot of work I'd rather not sign up for. If the configuration variant of the methods isn't there, we get a more useful error message. The problem here is b/c the result builder infra cant resolve a method to handle the optional, the type checker looks for another method that would work, which then finds the `configure` variation.

Reply:
Ohhh I see the issue... Hm...

We should probably add a comment here why this is useful for us since it's not obvious.

Out of curiosity, does providing the parameter name for the builder resolve the compiler error as well?

Reply:
I want to say I've seen this ambiguous error when I get syntax wrong inside the closure such as mislabeled/missing parameter. Removing `configure` seems like a good idea to me long term, but a huge undertaking for now. What about deprecating it to discourage future use and encourage refactoring existing uses?

Reply:
There's thousands upon thousands of usages of the configure syntax now – deprecating it is going to make a lot of fairly useless noise.

Reply:
> Out of curiosity, does providing the parameter name for the builder resolve the compiler error as well?

Yessir